packages/shopping/src/__tests__/acceptance/user.controller.acceptance.ts

```diff
@@ -3,8 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Client, expect, toJSON} from '@loopback/testlab';
-import {Response} from 'superagent';
+import {Client, expect /*, toJSON*/} from '@loopback/testlab';
```

Comment:
If `toJSON` is not used, we should just remove it.

Reply:
@raymondfeng , good catch. Just a silly comment I meant to leave temporarily while I was refactoring.